python/cudf/dataframe/series.py

```diff
@@ -133,6 +135,17 @@ def ndim(self):
         """
         return 1
 
+    @property
+    def name(self):
+        """Returns name of the Series.
+        """
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name
+        self._column._name = name
```

Comment:
Should use the column name property setter here instead of directly modifying the attribute.